xml/admin_monitoring_alerting.xml

```diff
@@ -198,6 +200,56 @@ placement:
         </step>
       </procedure>
     </sect2>
+
+    <sect2 xml:id="monitoring-custom-images">
+      <title>Deploying Using Custom or Local Images</title>
+      <para>
+        To deploy custom or local container images, the images need to be set
+        in &cephadm;. To do so, you will need to run the following command:
+      </para>
+      <screen>&prompt.cephuser;ceph config set mgr mgr/cephadm/<replaceable>option_name</replaceable> <replaceable>value</replaceable></screen>
+      <para>
+        Where <replaceable>option_name</replaceable> is any of the following names:
+      </para>
+
+      <itemizedlist>
+        <listitem><para>container_image_prometheus</para></listitem>
+        <listitem><para>container_image_node_exporter</para></listitem>
+        <listitem><para>container_image_alertmanager</para></listitem>
+        <listitem><para>container_image_grafana</para></listitem>
+      </itemizedlist>
+      <para>
+        By default, the following images are used as <literal>value</literal>:
+      </para>
+      <itemizedlist>
+        <listitem><para>registry.suse.com/caasp/v4.5/prometheus-server:2.18.0</para></listitem>
+        <listitem><para>registry.suse.com/caasp/v4.5/prometheus-node-exporter:0.18.1</para></listitem>
+        <listitem><para>registry.suse.com/caasp/v4.5/prometheus-alertmanager:0.16.2</para></listitem>
+        <listitem><para>registry.suse.com/ses/7/ceph/grafana:7.0.3</para></listitem>
+      </itemizedlist>
+      <para>For example:</para>
+      <screen>&prompt.cephuser;ceph config set mgr mgr/cephadm/container_image_prometheus prom/prometheus:v1.4.1</screen>
+      <note>
+        <para>
+          By setting a custom image, the default value will be overridden (but
+          not overwritten). The default value changes when updates become
+          available. By setting a custom image, you will not be able to update
+          the component you have set the custom image for automatically. You
+          will need to manually update the configuration (image name and tag) to
+          be able to install updates.
+        </para>
+        <para>
+          If you choose to go with the recommendations instead, you can reset
+          the custom image you have set before. After that, the default value
+          will be used again. Use <command>ceph config rm</command> to reset the
+          configuration option:
+        </para>
+        <screen>&prompt.cephuser;ceph config rm mgr mgr/cephadm/<replaceable>option_name</replaceable></screen>
```

Comment:
```suggestion
<screen>&prompt.cephuser;ceph config rm mgr mgr/cephadm/<replaceable>option_name</replaceable></screen>
```

Reply:
There isn't a difference in the result of the document when the indentation is not consistent. Is there another reason for suggesting to not having a consistent indentation?

Reply:
We always shift <screen>'s to the very left, our XMLformat config file is setup that way, just a professional deformaiton :-)